step.yml

```diff
@@ -1,9 +1,27 @@
 title: Carthage
 summary: |-
-  Runs selected Carthage command.
+  Runs the selected Carthage command.
 description: |-
-  Runs selected Carthage command. The step will cache your dependencies only when using `bootstrap` as `carthage_command` and you have `cache-pull` and `cache-push` steps in your workflow.
-  For more information about Carthage, visit the [Carthage GitHub page](https://github.com/Carthage/Carthage).
+  Downloads and builds the dependencies your Cocoa application needs.
+  
+  ### Configuring the Step
+  1. Add the **Carthage** Step in your Workflow.
+  2. Select `bootstrap` Carthage command in the **Carthage command to run** input. Make sure you have the **Bitrise.io Cache:Pull** Step before and the **Bitrise.io Cache:Push** Step after the **Carthage** Step in your Workflow to cache files and speed up your Bitrise build.
+  2. Provide your GitHub credentials in the **GitHub Personal Access Token** input. Don't worry, your credentials are safe with us since we store them encrypted and do not print them out in build logs.
```

Comment:
Might worth to mention that it is recommended for avoiding Github rate limit issue: https://github.com/bitrise-steplib/steps-carthage/pull/58/files#diff-91d8d269a1c9bd3631ddb04b80b01c4eL60